src/ru/skypro/Employee.java

```diff
@@ -0,0 +1,72 @@
+package ru.skypro;
+
+import java.util.Objects;
+
+public class Employee {
+    private static int forID = 1;
+    private int ID;
+    private String surName;
```

Comment:
переменные стоит именовать с маленькой буквы, статик переменные лучше отделять от обычных визуально пустой строкой
```suggestion
    private static int forId = 1;
    
    private int id;
    private String surName;
```